### DISCUSSION TITLE
Can we add conditions in the fields?

### DISCUSSION THREAD

**Discussion by CorentinBallaz** [grafana/grafana-json-datasource#109] [Q&A]
Hello,
First of all, thank you for this plugin, efficient and quick to use!
On my side, I'm going to look for data on an API in the following way:
[{"customer":"A","goods":"tables","delivery":"14-04-2021"},{"customer":"A","goods":"chairs","delivery":"14-04-2021"},{"customer":"A","goods":"tables","delivery":"15-04-2021"},{"customer":"B","goods":"bench","delivery":"17-04-2021"},...].
So I would like a customer variable, a commodity variable and a delivery variable in this order. Let's say I select customer A, I want the values in goods to be only tables and chairs (no bench). So I want to have a progressive condition, something that can be done in general on other data sources. How do I do this with this plugin?
I don't have access to modify the API...
Thank you in advance for your answer.

**Comment by marcusolsson** [comment]
This sounds like you could use [Filters](https://marcus.se.net/grafana-json-datasource/jsonpath#filters)?

Something like (haven't tested this): 

```
$.[?(@.goods == 'tables')].customer
```
